tests/backends/oracle/tests.py

```diff
@@ -69,6 +70,57 @@ def test_check_database_version_supported(self, mocked_get_database_version):
             connection.check_database_version_supported()
         self.assertTrue(mocked_get_database_version.called)
 
+    @unittest.skipUnless(connection.is_pool, "Pool specific tests")
+    def test_pool_set_to_true(self):
+        test_pool = connection.settings_dict["OPTIONS"]["pool"]
+        self.assertTrue(test_pool)
+
+    @unittest.skipUnless(connection.is_pool, "Pool specific tests")
+    def test_pool_reuse(self):
+        new_connection = connection.copy()
+        self.assertIsNotNone(new_connection.pool)
+
+        connections = []
+
+        def get_connection():
+            # copy() reuses the existing alias and as such the same pool.
+            conn = new_connection.copy()
+            conn.connect()
+            connections.append(conn)
+            return conn
+
+        try:
+            connection_1 = get_connection()  # First connection.
+            get_connection()  # Get the second connection.
+            sql = "select sys_context('userenv', 'sid') from dual"
```

Comment:
I think this should check the SID & serial # combination is reused: something like `select sid || ' ' || serial# from v$session where sid = sys_context('USERENV','SID')`

Reply:
To query v$session one requires a sysdba connection . Also to verify connection reuse in oracledb similar query is used.
https://github.com/oracle/python-oracledb/blob/main/tests/test_2400_pool.py#L760